src/context/DeleteMessage.ts

```diff
@@ -16,11 +16,11 @@ export class DeleteMessage extends CtxMenu
         });
     }
 
-    async run(int: ContextMenuInteraction)
+    async run(int: ContextMenuCommandInteraction)
     {
         const err = () => int.reply({ ...useEmbedify("Can't delete this message", settings.embedColors.error), ephemeral: true });
 
-        if(!int.isMessageContextMenu())
+        if(!int.isMessageContextMenuCommand())
```

Comment:
Doesn't it have to be `int.type === enum`?

Reply:
no idea. this is a function and it makes sense

Reply:
dwayne johnson squint gif

Reply:
did you test it by any chance?

Reply:
bruh, i told you in the discord i did lmfao

Reply:
i didn't run the code but it does assert the type correctly